WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -141,6 +141,16 @@ def results_submitted(competition, results_submission, submitter_user)
     @competition.uploaded_jsons.delete_all
   end
 
+  def registration_reminder(competition, user)
+    @competition = competition
+    @user = user
+    mail(
+      to: user.email,
+      reply_to: competition.organizers.pluck(:email),
+      subject: "#{competition.name} registration opens soon",
```

Comment:
This is a pretty public facing feature, I think we should probably i18n it from the start.

Reply:
Done.